projects/luajit/build.sh

```diff
@@ -61,7 +63,7 @@ for f in $(find build/tests/ -name '*_test' -type f);
 do
   name=$(basename $f);
   module=$(echo $name | sed 's/_test//')
-  corpus_dir="corpus/$module"
+  corpus_dir="corpus/corpus/$module"
```

Comment:
corpus_dir="corpus/corpus/$module"
```suggestion
  corpus_dir="corpus/corpus/${module}_test"
```

Reply:
Fixed:

```diff
--- a/projects/luajit/build.sh
+++ b/projects/luajit/build.sh
@@ -62,7 +62,6 @@ cmake --build build --parallel
 for f in $(find build/tests/ -name '*_test' -type f);
 do
   name=$(basename $f);
-  module=$(echo $name | sed 's/_test//')
   corpus_dir="corpus/corpus/$module"
   echo "Copying for $module";
   cp $f /
```

Reply:
I'd prefer proposed change:
1. $module is undefined here
2. Even if defined, it would be with '_test' suffix, so it would require renaming of all corpuses and dictionaries in every toml file.

Reply:
Updated:

```diff
--- a/projects/luajit/build.sh                                                                                                                                
+++ b/projects/luajit/build.sh                                                 
@@ -62,7 +62,8 @@ cmake --build build --parallel                                                                                                              
 for f in $(find build/tests/ -name '*_test' -type f);      
 do                                                                            
   name=$(basename $f);                                                        
-  corpus_dir="corpus/corpus/$module"                                                                                                                         
+  module=$(echo $name | sed 's/_test//')                                      
+  corpus_dir="corpus/corpus/${module}_test"                       
   echo "Copying for $module";                                                                                                                                
   cp $f /                                                                                                                                                    
   [[ -e $corpus_dir ]] && cp -r $corpus_dir /corpus_$module 
```